server/core/src/irods_signal.cpp

```diff
@@ -46,26 +48,26 @@ void stacktrace_signal_handler(int _signal)
             *msec_end = '.';
 
             if (auto [pid_end, ec] = std::to_chars(msec_end + 1, buf_end, getpid()); std::errc{} == ec) {
-                // Terminate the hidden file path string. The filename should have the following format:
+                // Terminate the path string. The filename should have the following format:
                 //
                 //     <epoch_seconds>.<remaining_milliseconds>.<agent_pid>
                 //
                 // This keeps the files sorted by timestamp and by pid.
                 *pid_end = 0;
 
-                char file_path[max_path_length];
-                std::strcpy(file_path, file_path_hidden);
+                char file_path[_POSIX_PATH_MAX];
+                std::strcpy(file_path, file_path_unprocessed);
                 
                 // An indicator to the thread handling the logging of these files that this file
                 // is not ready to be logged (i.e. the signal handler is still writing to it).
-                std::strcat(file_path_hidden, ".not_ready_for_logging");
+                std::strcat(file_path_unprocessed, ".not_ready_for_logging");
```

Comment:
should ".not_ready_for_logging" be a variable?   it's used elsewhere... the processor, I presume...

`unprocessed_stacktrace_file_suffix`?

Reply:
yes it should.

Reply:
i haven't figured out where though

Reply:
i'll put it in the header file. it is specific to this, so it makes sense.

Reply:
done.